backend/app/tests/core/test_security.py

```
@@ -192,4 +192,4 @@ def test_generate_creates_verifiable_key(self, db: Session):
         auth_context = APIKeyManager.verify(db, api_key_response.key)
 
         assert auth_context is not None
-        assert auth_context.user.id == api_key_response.user_id
+        assert auth_context.user.id == api_key_response.user_id
```

Ruff (0.14.10)

No newline at end of file

Add trailing newline

(W292)